doc/release-notes/0.46/0.46.md

```diff
@@ -62,21 +62,59 @@ The following table covers notable changes in v0.46.0. Further information about
-<td valign="top">A new option <tt>-XX:[+|-]EnableExtendedHCR</tt> is added.</td>
+<td valign="top">The extended Hot Code Replace (HCR) capability is disabled by default and the <tt>-XX:[+|-]EnableExtendedHCR</tt> option is added to enable or disable the HCR capability.</td>
 <td valign="top">All versions</td>
-<td valign="top">By default, the extended Hot Code Replace (HCR) capability in the VM is disabled for all OpenJDK versions. You can enable or disable the HCR capability by using the <tt>-XX:[+|-]EnableExtendedHCR</tt> option.
+<td valign="top">By default, the extended HCR capability in the VM is disabled for all OpenJDK versions. You can enable the HCR capability by using the <tt>-XX:+EnableExtendedHCR</tt> option.
 
 The extended HCR feature is deprecated in this release and will be removed in a future release. From OpenJDK 25 onwards, extended HCR will not be supported. Following that, the extended HCR support will be removed from other earlier OpenJDK versions also.
 </td>
 </tr>
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/pull/19245">#19245</a>, <a href="https://github.com/eclipse-openj9/openj9/pull/19165">#19165</a></td>
+<td valign="top">A new system property <tt>-Dcom.ibm.tools.attach.retry</tt> is added to improve <tt>jcmd</tt> attaching in case of the <tt>SocketException</tt> error on Windows&trade; platform.</td>
+<td valign="top">All versions (Windows)</td>
+<td valign="top">When the <tt>jcmd</tt> tool sends a command to a running VM, the command might throw the <tt>Socket Exception</tt> error on Windows platform. Instead of failing the attaching request, you can specify the number of times the tool retries attaching to the target VM with the new system property, <tt>-Dcom.ibm.tools.attach.retry</tt>.
+</td>
+</tr>
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/pull/19070">#19070</a></td>
+<td valign="top">Core allocation cache statistics per thread is included in the <tt>-Xtgc:allocation</tt> report.</td>
+<td valign="top">All versions</td>
+<td valign="top">The <tt>-Xtgc:allocation</tt> option prints thread-specific allocation cache (TLH) statistics in addition to the cumulative allocation statistics.
+</td>
+</tr>
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/pull/19323">#19323</a>, <a href="https://github.com/eclipse-openj9/openj9/pull/19442">#19442</a></td>
+<td valign="top">The <tt>-XX:[+|-]ShareOrphans</tt> option is added to control storing of orphan classes in the shared classes cache.</td>
+<td valign="top">All versions</td>
+<td valign="top">When <tt>-Xshareclasses</tt> was specified, only those class loaders that implemented the OpenJ9's public shared classes cache APIs (and its child class loaders) could store classes to the shared classes cache. Custom class loaders that did not implement these cache APIs cannot pass the module or class path information to the VM. Classes of such class loaders were not stored to the cache.
```

Comment:
Minor tense changes: 

was specified, only *the* class loaders that implemented the OpenJ9's public shared classes cache APIs (and its child class loaders) could store classes to the shared classes cache. Custom class loaders that did not implement these cache APIs *couldn't* pass the module or class path information to the VM. Classes of such class loaders were not stored to the cache.

*Now, users* can enable class sharing from all class loaders, irrespective of whether the class loader implements the shared classes cache API, with the `-XX:+ShareOrphans` option.

Reply:
We avoid using the term "users". I will change the sentence to "Now, you can enable..."